javers-core/src/main/java/org/javers/repository/jql/QueryBuilder.java

```diff
@@ -508,13 +508,38 @@ public QueryBuilder withCommitPropertyLike(String name, String valueFragment){
 
     /**
      * Only snapshots with a given version.
+     *
+     * @see #fromVersion(long)
+     * @see #toVersion(long)
      */
     public QueryBuilder withVersion(long version) {
         Validate.argumentCheck(version > 0, "Version is not a positive number.");
         queryParamsBuilder.version(version);
         return this;
     }
 
+    /**
+     * All snapshots from or equals to a given version.
+     * @see #withVersion(long)
+     * @see #toVersion(long)
+     */
+    public QueryBuilder fromVersion(long fromVersion) {
+        Validate.argumentCheck(fromVersion > 0, "Version is not a positive number.");
+        queryParamsBuilder.fromVersion(fromVersion);
+        return this;
+    }
+
+    /**
+     * All snapshots up to (exclusive) a given version.
+     * @see #withVersion(long)
+     * @see #fromVersion(long)
+     */
+    public QueryBuilder toVersion(long toVersion) {
```

Comment:
Making fromVersion inclusive but toVersion exclusive, seems counter intuitive for me. Could you please change toVersion also to be inclusive (so from to would work like the BETWEEN operator in SQL)